addons/website_sale/controllers/combo_configurator.py

```diff
@@ -88,6 +88,7 @@ def website_sale_combo_configurator_update_cart(
                     ],
                     linked_line_id=values['line_id'],
                     combo_item_id=combo_item['combo_item_id'],
+                    line_id=False
```

Comment:
I believe this is missing a trailing comma.

Small additional nit: I'd move this just below the `product_id=...` line (to maintain the same args ordering as the method signature).

Reply:
Not sure how I missed that in the commit, thank you! 

All done.